components/elm/src/main/elm_driver.F90

```diff
@@ -175,6 +175,24 @@ module elm_driver
   use elm_varctl                  , only : budget_ann, budget_ltann, budget_ltend
 
   use timeinfoMod
+#if defined(CLDERA_PROFILING)
+   use iso_c_binding, only: c_loc
+   use cldera_interface_mod, only: cldera_init, cldera_set_log_unit, &
+                                   cldera_set_masterproc, max_str_len, &
+                                   cldera_add_partitioned_field, &
+                                   cldera_set_field_part_extent, &
+                                   cldera_set_field_part_data, &
+                                   cldera_commit_all_fields,   &
+                                   cldera_commit_field, cldera_switch_context, &
+                                   cldera_compute_stats
+!    use physics_buffer,   only: physics_buffer_desc, col_type_grid, pbuf_get_index, &
+!                                pbuf_get_field_rank, pbuf_get_field_dims, pbuf_get_field, &
+!                                pbuf_get_field_name, pbuf_has_field, pbuf_get_field_persistence, &
+!                                persistence_global
+!    use ppgrid,           only: begchunk, endchunk, pcols, pver
+!    use phys_grid,        only: get_ncols_p, get_gcol_all_p, get_area_all_p
+!    use constituents,     only: pcnst, cnst_name
```

Comment:
delete?

Reply:
deleted all except the stats and context switch